kuma.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: kuma
+  version: 2.8.2
+  epoch: 0
+  description: A multi-zone service mesh for containers, Kubernetes and VMs.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kumahq/kuma
+      tag: ${{package.version}}
+      expected-commit: ffb0a135b8320cd474c503b785b43627d77a1338
+
+subpackages:
+  - name: kuma-cp
+    pipeline:
+      - uses: go/build
+        with:
+          packages: ./app/kuma-cp
+          output: kuma-cp
+      - uses: strip
+    test:
+      pipeline:
+        - name: Verify kuma-cp
+          runs: |
+            kuma-cp --help
+
+update:
+  enabled: true
+  github:
+    identifier: kumahq/kuma
+    use-tag: true
```

Comment:
I don't think this line is required - the upstream repo seems to use releases!

Reply:
Might need tags because there are multiple versions supported? or will the bot pick up on that and only update this with the correct version

Reply:
Yup, the bot should pick up the latest tag.